setup.py

```diff
@@ -16,19 +16,31 @@
 
 from setuptools import setup
 
-with open('README.md', 'r', encoding='utf-8') as file:
+with open("README.md", "r", encoding="utf-8") as file:
     long_description = file.read()
 
-setup(name='david-energy-openleadr',
-      version='0.5.33',
-      description='Python3 library for building OpenADR Clients (VENs) and Servers (VTNs)',
-      long_description=long_description,
-      long_description_content_type='text/markdown',
-      url='https://openleadr.org',
-      project_urls={'GitHub': 'https://github.com/openleadr/openleadr-python',
-                    'Documentation': 'https://openleadr.org/docs'},
-      packages=['openleadr', 'openleadr.service'],
-      python_requires='>=3.7.0',
-      include_package_data=True,
-      install_requires=['xmltodict==0.13.0', 'aiohttp>=3.8.3,<4.0.0', 'apscheduler>=3.10.0,<4.0.0', 'jinja2>=3.1.2,<4.0.0', 'signxml>=3.1.0,<3.2.0'],
-      entry_points={'console_scripts': ['fingerprint = openleadr.fingerprint:show_fingerprint']})
+setup(
+    name="david-energy-openleadr",
+    version="0.5.34",
+    description="Python3 library for building OpenADR Clients (VENs) and Servers (VTNs)",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    url="https://openleadr.org",
+    project_urls={
+        "GitHub": "https://github.com/openleadr/openleadr-python",
+        "Documentation": "https://openleadr.org/docs",
+    },
+    packages=["openleadr", "openleadr.service"],
+    python_requires=">=3.7.0",
+    include_package_data=True,
+    install_requires=[
+        "xmltodict==0.13.0",
+        "aiohttp>=3.8.3,<4.0.0",
+        "apscheduler>=3.10.0,<4.0.0",
+        "jinja2>=3.1.2,<4.0.0",
+        "signxml>=3.1.0,<3.3.0",
```

Comment:
The change seems good. Just curious re: how we're going to test? Any tests events we can play?

Reply:
🤔  not sure

Reply:
@tedsmoore any thoughts in how we might test changes to the OpenADR implementation? I see that we are still getting regular pings to the `/api/OpenADR2/Simple/EiEvent` endpoint.